portal/development_gulpfile.js

```diff
@@ -193,6 +195,28 @@ const orgTreeViewLess = function(callback) {
 };
 exports.orgTreeViewLess = series(orgTreeViewLess);
 
+/*
+ * transforming exercise diet less to css
+ */
+const exerciseDietLess = function(callback) {
+    console.log("Compiling Exercise & Diet Less...");
+    src(lessPath + "/" + ED + ".less")
+        .pipe(sourcemaps.init({
+            sources: [lessPath + "/" + ED + ".less"]
+        }))
+        .pipe(less({
+            plugins: [cleancss]
+        }))
+        .pipe(postCSS())
+        .pipe(sourcemaps.write("../../../../"+mapPath)) /* note to write external source map files, pass a path relative to the destination */
```

Comment:
I'm not sure what directory this is relative to (ie the directory it's run in).

Are you able to make it relative to an existing `fooPath` variable?